google/cloud/alloydb/connector/client.py

```diff
@@ -129,9 +129,15 @@ async def _get_metadata(
         resp = await self._client.get(url, headers=headers, raise_for_status=True)
         resp_dict = await resp.json()
 
+        # Remove trailing period from PSC DNS name.
+        psc_dns = resp_dict.get("pscDnsName")
+        if psc_dns:
+            psc_dns = psc_dns.rstrip(".")
```

Comment:
Was this necessary in Python?

Reply:
I tried without it and it failed: 

```
>       self._sslobj.do_handshake()
E       ssl.SSLError: [SSL: SSLV3_ALERT_UNEXPECTED_MESSAGE] sslv3 alert unexpected message (_ssl.c:1131)
```

https://github.com/GoogleCloudPlatform/alloydb-python-connector/actions/runs/8471675334/job/23489146452